lib/new-flatfile.ts

```diff
@@ -72,3 +73,35 @@ export const addGuestToSpace = async ({
     return null;
   }
 };
+
+export const listWorkbooks = async ({ spaceId }: { spaceId: string }) => {
+  try {
+    const flatfile = flatfileClient();
+
+    const result = await flatfile.workbooks.list(
+      spaceId as ListWorkbooksRequest
+    );
+
+    console.log("list now");
+
+    return result;
+  } catch (e) {
+    console.log("error", JSON.stringify(e, null, 2));
+    return null;
+  }
+};
+
+export const getRecords = async ({ sheetId }: { sheetId: string }) => {
+  try {
+    const flatfile = flatfileClient();
+
+    const result = await flatfile.records.get(sheetId);
+
+    console.log("getRecords result", result);
+
+    return result;
```

Comment:
result or result.data here? What are we actually returning?

Reply:
The point of the req is to get records. I should dig into the response and actually return records.